jobs/k8s-e2e-external-storage.yaml

```diff
@@ -25,11 +36,11 @@
       lightweight-checkout: true
     triggers:
       - github-pull-request:
-          status-url: ${RUN_DISPLAY_URL}
-          status-context: ci/centos/k8s-e2e-external-storage
+          status-url: $RUN_DISPLAY_URL
```

Comment:
@nixpanic we don't need `{}` here?

Reply:
looks like its not required and deliberately dropped. But lets get confirmed from Niels,  The change looks good to me.

Reply:
Yes, dropped because this is ia job-template. In the template variables are expressed like `{var_name}` and this causes a failure with `${RUN_DISPLAY_URL}`, it gets interpreted as `{RUN_DISPLAY_URL}` with missing `RUN_DISPLAY_URL` variable.